media/css/mozorg/home/home-promo.less

```diff
@@ -890,6 +890,29 @@ html[lang|="en"] {
 }
 
 
+/***************************************************************************/
+// Thimble
+.promo-small-landscape.thimble {
+
+    background-color: #2D9961;
+
+    h2 {
+        .at2x('/media/img/home/panel/thimble/mozilla-thimble.png', 69px, 75px);
+        background-repeat: no-repeat;
+        margin: 30px 20px;
+        padding: 8px 0 0 85px;
+        height: 75px;
+    }
+}
+
+.html-rtl {
+    h2 {
```

Comment:
Hah, I had it like that and then changed it. Totally agree, targeting all h2 when it is rtl is not a good idea ;p